.appveyor.yml

```diff
@@ -10,13 +10,10 @@ environment:
     secure: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 
   matrix:
-    - CONFIG: win_clang_variantcling_6.14.06cxx_compilervs2015vc14
+    - CONFIG: win_cxx_compilervs2008
```

Comment:
I think you should remove this part of the matrix as well as `.ci_support/win_cxx_compilervs2008.yaml` as they're not supposed to be built according to the skip.

(due to the ugly hack I suggested on Gitter.)